sdk/go/aws/apigateway/stage.go

```diff
@@ -156,7 +156,7 @@ type stageState struct {
 	// The identifier of a client certificate for the stage.
 	ClientCertificateId *string `pulumi:"clientCertificateId"`
 	// The ID of the deployment that the stage points to
-	Deployment *string `pulumi:"deployment"`
+	Deployment interface{} `pulumi:"deployment"`
```

Comment:
This looks a bit concerning, less precise type or is intentional?

Reply:
This is intentional--the original type was incorrect with respect to the actual types of the state inputs.